cmd/speep/main.go

```diff
@@ -24,52 +24,81 @@ func main() {
 }
 
 func run() error {
+
+	const usage = `Usage of speep: speep <command> <flag1> ... <flag2> ...
+	
+	Command Options:
+		get [--all] | [--name <name>]
+		add --name <name> --key <key>
+		delete  [--all] | [--name <name>]
+
+	Flag Options:
+	  -a, --all retrieve all shortcuts
+	  -n, --name name of the registered shortcut key: Use \"\" for more than one word e.g. -name \"to the back of the line\".
+	  -k, --key registered shortcut key`
+
+	const longFlagName = "Use \"for more than one word e.g. --name \"to the back of the line\""
+	const shortFlagName = "Use \"for more than one word e.g. -n \"to the back of the line\""
```

Comment:
Use backtick-strings to avoid these `\` escapes and improve clarity:

```suggestion
	const longFlagName = `Use double quotes ("") for more than one word e.g. --name "to the back of the line"`
	const shortFlagName = `Use double quotes ("") for more than one word e.g. -n "to the back of the line"`

```

Reply:
Applies throughout